templates/Makefile

```diff
@@ -22,6 +22,11 @@ apply-env:
 	terraform init && \
 	terraform apply $(AUTO_APPROVE)
 
+pre-k8s:
```

Comment:
This requires `update-k8s-conf` to be run to set up kubeconfig. Maybe remove it from the call on line 30 and add it after `apply-env` on line 4?

Reply:
done